api/provider.go

```diff
@@ -15,17 +15,17 @@ type Provider struct {
 
 type CreateProviderRequest struct {
 	Name         string `json:"name" validate:"required" example:"okta"`
-	URL          string `json:"url" validate:"required" example:"infrahq.okta.com"`
-	ClientID     string `json:"clientID" example:"0oapn0qwiQPiMIyR35d6"`
-	ClientSecret string `json:"clientSecret" example:"jmda5eG93ax3jMDxTGrbHd_TBGT6kgNZtrCugLbU"`
+	URL          string `json:"url" validate:"fqdn,required" example:"infrahq.okta.com"`
```

Comment:
Can this field accept a port and/or path?

Reply:
I tested it and it's a bit wonky, removed `fqdn`